receivable.go

```diff
@@ -126,13 +126,21 @@ func (t *receivable) handleOrClose(p pdu.PDU) (closing bool) {
 
 		default:
 			var responded bool
-			if p.CanResponse() && t.settings.response != nil {
+			if !t.settings.DisablePDUAutoResponse && p.CanResponse() && t.settings.response != nil {
 				t.settings.response(p.GetResponse())
 				responded = true
 			}
 
 			if t.settings.OnPDU != nil {
 				t.settings.OnPDU(p, responded)
+				if p.CanResponse() && responded == false {
+					response := p.GetCustomResponse()
+					if response != nil {
+						t.settings.response(*response)
+					} else {
+						t.settings.response(p.GetResponse())
+					}
+				}
```

Comment:
I think what do you want here is custom response, and you prefer it.

In this way, you don't need `DisablePDUAutoResponse`.

Just change the source code into:
```go
if p.CanResponse() && t.settings.response != nil {
   // Change internal logic
   response := p.GetCustomResponse()
   if response != nil {
        t.settings.response(*response)
   } else {
       t.settings.response(p.GetResponse())
   }
   responded = true
}

// Keep this
if t.settings.OnPDU != nil {
    t.settings.OnPDU(p, responded)
}
```

Reply:
As you can see, we don't need dangerous setting `DisablePDUAutoResponse`

All we need to do is changing the logic of how we get the response.